content/blog/2019-08-15-new-react-devtools.md

```diff
@@ -4,7 +4,7 @@ author: [bvaughn]
 ---
 We are excited to announce a new release of the React Developer Tools, available today in Chrome, Firefox, and (Chromium) Edge!
```

Comment:
Translate this to hindi

Ex: `हम Chrome, Firefox और (Chromium) Edge में आज उपलब्ध React डेवलपर टूल्स की एक नई रिलीज़ की घोषणा करने के लिए उत्साहित हैं!`

Similar to this, complete file needs to be translated.